docs/README.md

```diff
@@ -1,6 +1,6 @@
 # php-src docs
 
-This is the home of the php-src internal documentation. It is in very early stages, but is intended
+This is the home of the php-src internal documentation hosted at [php.github.io/php-src/](https://php.github.io/php-src/). It is in very early stages, but is intended
```

Comment:
```suggestion
This is the home of the php-src internal documentation, hosted at [php.github.io/php-src/](https://php.github.io/php-src/). It is in very early stages, but is intended
```

Please also respect the max character length of 100.

Reply:
addressed